crates/store/Cargo.toml

```diff
@@ -42,11 +44,20 @@ tracing                = { workspace = true }
 
 [dev-dependencies]
 assert_matches        = { workspace = true }
+criterion             = { version = "0.5" }
+fs-err                = { workspace = true }
 miden-lib             = { features = ["testing"], workspace = true }
 miden-node-test-macro = { workspace = true }
 miden-node-utils      = { features = ["testing", "tracing-forest"], workspace = true }
 miden-objects         = { default-features = true, features = ["testing"], workspace = true }
-pretty_assertions     = { workspace = true }
 rand                  = { workspace = true }
 regex                 = { version = "1.11" }
 termtree              = { version = "0.5" }
+
+[features]
+default = []                       # rocksdb has just too many requirements on CI for now
+rocksdb = ["miden-crypto/rocksdb"]
```

Comment:
We are not using the `rocksdb` backend yet, right? If so, it may make sense to remove the feature from this PR (and this may also roll back some Makefile changes).